packages/blockchain-ui/src/components/pages/TutorialSinglePage/TutorialSinglePage.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FunctionComponent } from 'react';
+import './TutorialSinglePage.scss';
```

Comment:
This can be removed as it doesn't exist!

Reply:
oops I removed the file but forgot the import...